includes/namespace.php

```diff
@@ -136,14 +140,15 @@ function remove_site( $error, $site ) {
 	}
 
 	$injector = Services::get_injector();
+	// If something went wrong with the injector, exit early.
 	if ( ! method_exists( $injector, 'make' ) ) {
 		return;
 	}
-	$story = $injector->make( Story_Post_Type::class );
+	$capabilities = $injector->make( User\Capabilities::class );
 
 	$site_id = (int) $site->blog_id;
 	switch_to_blog( $site_id );
-	$story->remove_caps_from_roles();
+	$capabilities->add_caps_to_roles();
```

Comment:
This is the `remove_site()` function. I assume you want to _remove_ caps, not _add_ caps?

```suggestion
	$capabilities->remove_caps_from_roles();
```

Reply:
@spacedmonkey ^